### ISSUE TITLE
bug: Sidemenu is inaccessible with VoiceOver in IOS

### ISSUE THREAD

**Issue by ethier** [driftyco/ionic#4099]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>In iOS 8, with VoiceOver enabled, the main menu doesn’t receive focus when opened, but only after the user has navigated to the bottom of the page.

Steps to reproduce:

* In iOS 8 enable VoiceOver in the Accessibility menu
Open the test app
* Single-finger swipe right until you focus on the “menu” icon
* Double tap the screen to open the menu
* Swipe right again. You’ll notice that the focus continues down the home page content, rather than inside the main menu
* Continue swiping right until you access the menu

Expected behaviour:

When the menu is opened, the focus must be shifted to the first menu item, allowing the user to continue swiping to navigate through the menu items. When the user swipes past the last menu item, the focus must be placed back at the first menu item. To close the menu the user must be able to double tap outside the menu.

Notes:
Tested on iOS 8.4, iPhone 5S and iPhone 5C.

Given that this is an issue in the native app, I've created a [repo](https://github.com/ethier/ionic-accessibility) to clone that highlights the issue identified (no codepen).

With the latest ionic installed, clone the [repo](https://github.com/ethier/ionic-accessibility), add the IOS platform and deploy to a device (VoiceOver doesn't work in an emulator) to re-create this issue.</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @ethier!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen this issue by creating a new one, and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.
